reference/docs-conceptual/dsc/configurations/compositeConfigs.md

```diff
@@ -46,11 +46,15 @@ Configuration NestedFileConfig
 }
 ```
 
-In this example, `FileConfig` takes two mandatory parameters,  **CopyFrom** and **CopyTo**,
-which are used as the values for the **SourcePath** and **DestinationPath** properties in the `File` resource block.
-The `NestedConfig` configuration calls `FileConfig` as if it were a resource.
-The properties in the `NestedConfig` resource block (**CopyFrom** and **CopyTo**) are the parameters of the `FileConfig` configuration.
+In this example, `FileConfig` takes two mandatory parameters, **CopyFrom** and **CopyTo**, which are
+used as the values for the **SourcePath** and **DestinationPath** properties in the `File` resource
+block. The `NestedConfig` configuration calls `FileConfig` as if it were a resource. The properties
+in the `NestedConfig` resource block (**CopyFrom** and **CopyTo**) are the parameters of the
+`FileConfig` configuration.
+
+DSC doesn't currently support nesting configurations within nested configurations. You can only
+nest a configuration one layer deep.
 
 ## See Also
 
-- [Composite resources--Using a DSC configuration as a resource](../resources/authoringResourceComposite.md)
\ No newline at end of file
+* [Composite resources--Using a DSC configuration as a resource](../resources/authoringResourceComposite.md)
```

Comment:
Use hyphen for bullets instead of asterisk - Avoids confusion with italics or bold markup.

Reply:
That's right! I changed it because I thought it was reversed! fixed

Reply:
done